app/javascript/shared/components/__tests__/UserStore.test.js

```diff
@@ -0,0 +1,79 @@
+import fetch from 'jest-fetch-mock';
+import { UserStore } from '../UserStore';
+
+global.fetch = fetch;
+
+function fakeUsers() {
+  return JSON.stringify([
+    { name: 'Alice', username: 'alice', id: 1 },
+    { name: 'Bob', username: 'bob', id: 2 },
+    { name: 'Charlie', username: 'charlie', id: 3 },
+    { name: 'Almost Alice', username: 'almostalice', id: 4 },
+  ]);
+}
+
+describe('UserStore', () => {
+  beforeEach(() => {
+    fetch.resetMocks();
```

Comment:
I really like this consistency of how you mock global fetch and clear things up in between